src/App.js

```diff
@@ -1,25 +1,44 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
```

Comment:
```suggestion
import { useState } from "react";
```

React is not required here. 

See: 

- https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html